plugins/c/hu.bme.mit.gamma.xsts.codegeneration.c/xtend-gen/hu/bme/mit/gamma/xsts/codegeneration/c/CodeBuilder.java

```diff
@@ -0,0 +1,497 @@
+/**
```

Comment:
This file is unnecessary; add a single .gitignore in xtend-gen with the above-specified content.